### ISSUE TITLE
JAWS does not read text in Visual Studio 2017 creation wizards

### ISSUE THREAD

**Issue by kastwey** [FreedomScientific/VFO-standards-support#17]
### Summary
JAWS 2018 doesn't read text in Visual studio 2017 creation wizards text fields 

### Stepts to reproduce:

1. Open Visual Studio 2017.
2. File / new / Project.
3. Tab until name text edit.
4. Write the project name and try to read the text box content by using arrows.

### Expected result

JAWS should read the text box content.-

### Actual result

JAWS read "blank" as the textbox is empty, what it's false.

#### JAWS version and build number
JAWS 2018.1710.42

 
#### Operating System and version
Windows 10 creators update

**Comment by stevefaulkner** [comment]
Hi @kastwey 
apologies, but this tracker is strictly for JAWS HTML/ARIA/SVG/CSS feature support issues.
Please contact [Freedom Scientific Technical Support](https://www.freedomscientific.com/Forms/TechSupport) to report general issues with JAWS.

**Note:** I talked with a FS engineer about the issue you raised and it is a known issue that is being worked on, so hopefully it will be fixed in a JAWS release in the near future.
